docs/docs/userguide/configuration/JIRA_CONFIGURATION.md

```diff
@@ -142,3 +155,11 @@ jira_db_master_username = "<DB_MASTER_USERNAME>"   # e.g. "postgres"
 ```terraform
 jira_db_master_password = "<DB_MASTER_PASSWORD>"   # default value is null
 ```
+### Shared Home Restore
+
+`jira_shared_home_snapshot_id` sets id of shared home EBS snapshot. This will spin up EBS volume and it will be mounted to the NFS server and used when the product is started.
```

Comment:
```suggestion
`jira_shared_home_snapshot_id` sets the id of the shared home EBS snapshot to use. This will spin up an EBS volume that will be mounted to the NFS server and used when the product is started.
```